microcosm_pubsub/chain/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class AttributeNotFound(Exception):
+    def __init__(self, parent, attribute):
+        super().__init__(f"Failed to find attribute `{attribute}` on `{parent}`")
```

Comment:
Worth adding a test for those error messages?

Reply:
Hrm, sure.